src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexWriter.cs

```diff
@@ -12,32 +12,30 @@
 //
 
 using System.Collections.Generic;
-using System.Collections;
 using System.Globalization;
 
 namespace System.Text.RegularExpressions
 {
     internal sealed class RegexWriter
     {
-        internal int[] _intStack;
-        internal int _depth;
-        internal int[] _emitted;
-        internal int _curpos;
-        internal Dictionary<string, int> _stringhash;
-        internal List<String> _stringtable;
-        // not used! internal int         _stringcount;
-        internal bool _counting;
-        internal int _count;
-        internal int _trackcount;
-        internal Dictionary<Int32, Int32> _caps;
-
-        internal const int BeforeChild = 64;
-        internal const int AfterChild = 128;
-
-        /*
-         * This is the only function that should be called from outside.
-         * It takes a RegexTree and creates a corresponding RegexCode.
-         */
+        private int[] _intStack;
+        private int _depth;
+        private int[] _emitted;
+        private int _curpos;
+        private readonly Dictionary<string, int> _stringhash;
+        private readonly List<String> _stringtable;
+        private bool _counting;
+        private int _count;
+        private int _trackcount;
+        private Dictionary<Int32, Int32> _caps;
+
+        private const int BeforeChild = 64;
+        private const int AfterChild = 128;
```

Comment:
All the fields in this class are only used inside this class (even in the desktop code) so they can be `private`.